Book.php

```diff
@@ -0,0 +1,143 @@
+<?php
```

Comment:
And the main thing why it's not working at all: In new mediawiki scheme with `wfLoadExtension` you have to specify which files to use, so you need to update `extension.json` file to add `Book.php` in the same way like other files are there.